README.md

```diff
@@ -38,6 +38,14 @@ in the P4_tutorial.pdf in the tutorial directory.
 
 A P4 Cheat Sheet is also available [online](https://drive.google.com/file/d/1Z8woKyElFAOP6bMd8tRa_Q4SA1cd_Uva/view?usp=sharing)
 which contains various examples that you can refer to.
+
+## P4 Documentation
+
+The documentation for P4_16, P4_14, P4 Runtime and Portable Switch Architecture(PSA) is available [here](https://p4.org/specs/)
```

Comment:
This statement is completely true, but it seems not very relevant to working on these exercises at the moment that P4_14 and Portable Switch Architecture documentation can be found there, because these exercises use P4_16, P4Runtime, and the v1model architecture only, at least last time I checked.